src/confcom/azext_confcom/security_policy.py

```diff
@@ -517,9 +547,58 @@ def populate_policy_content_for_all_images(
             for message in message_queue:
                 logger.warning(message)
 
+            out_images = list(filter(lambda image: image.get_layers(), self.get_images()))
+            self.set_images(out_images)
+
+    def should_eliminate_container_covered_by_fragments(self, image):
+        for fragment_image in self._fragment_contents:
+            # we're not comparing layers to save computation time
+            fragment_image["layers"] = []
+            # TODO: make this print a warning if there is a fragment image that's close
+            # TODO: is "latest" tag assumed by this point?
```

Comment:
not sure about the context of this comment, but does the latest tag TODO need to be addressed rn?

Reply:
I'll add stuff for this, I don't think it is assumed yet